### DISCUSSION TITLE
Tile-converter 3.0 3DTiles to I3S conversion Image issue

### DISCUSSION THREAD

**Discussion by james-1595** [visgl/loaders.gl#1850] [Q&A]
Hi,

I was trying to convert a 3DTILES which was exported by Cesium's 3D Tiling Pipeline, this is how it looks like for the 3DTILES itself:
![image](https://user-images.githubusercontent.com/91245705/134468450-e97df426-4bfb-4baa-99d8-bb5628ec4288.png)

I ran this command with the 3DTILES in a Ubuntu docker environment:
`npx tile-converter --input-type 3DTILES --tileset sample_tileset/tileset.json --output sample_tileset --name sample_export --slpk`

Here's how it looks at the same location of the model:
![image](https://user-images.githubusercontent.com/91245705/134469185-30abf383-3c81-4cdb-9f94-d7d67d5a703e.png)

Here's another conversion where the model had holes on it:
![image](https://user-images.githubusercontent.com/91245705/134476393-8a830b9d-3c1b-419f-aaf0-07e331dbad02.png)

While the 3DTILES that used to export the slpk looks fine:
![image](https://user-images.githubusercontent.com/91245705/134476579-44b9b072-7f3d-4584-8aca-b8f50181aa5a.png)

To me it Looks like the image interpolation during the process was the issue, any suggestions? I'm happy to provide test data to reproduce the issue.

Any help will be much appreciated! Thanks!

Here's the link to the log (it's too long that cannot paste here):
https://raw.githubusercontent.com/james-1595/tile-converter-issue/main/tile-converter-log.log

**Comment by Tamrat-B** [comment]
@james-1595  thank you for submitting the issue. The converter has been updated few times since this bug was filed. Can you please try with the latest version (3.1.8)? Also it will be great if you can provide a test data that shows the issue. Thx!
